playbooks/provisioner/foreman/cleanup.yaml

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
@tkammer What exactly did you decide about the work flow?
A user should run ir-provisioner forman --cleanup first?!?
I think we should discuss about it...